hap.coffee

```diff
@@ -112,6 +120,20 @@ module.exports = (env) =>
         env.logger.debug("unsupported device type: " + device.constructor.name)
         return null
 
+    createAccessoriesFromTemplate: (device) =>
```

Comment:
Agree but I had only half a Sunday and didn't want to mess to much with the architecture